File: reference.net/db4o-reference-chapters/src/com/db4odoc/lists/CollectionFactory.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */
using System;
using System.Collections;

namespace Db4objects.Db4odoc.Lists
{
	public class CollectionFactory
	{
		public static IList newList()
		{
			return  new VerboseList();
		}
	}
}

File: reference.vb.net/Db4objects.Db4odoc/Lists/CollectionFactory.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 

Imports System
Imports System.Collections

Namespace Db4objects.Db4odoc.Lists
    Public Class CollectionFactory
        Public Shared Function NewList() As IList
            Return New VerboseList()
        End Function
    End Class
End Namespace

